placeholder

```diff
@@ -0,0 +1 @@
+placeholder
```

Comment:
drop this and add the CRDValidationRatcheting gate to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Removed extraneous file. The gate is already listed in the page you linked